caesar_cipher/run.rb

```diff
@@ -0,0 +1,19 @@
+require_relative 'lib/cipher'
+
+status = true
+
+while status
+  print 'Please enter top secret text for encryption : '
+  user_message = gets.chomp
+  print 'Please enter number for encryption  : '
+  user_input_number = gets.chomp.to_i
+  if user_input_number >= 26 || user_input_number <= 0
```

Comment:
@zzokki81 remove this validation from here. Instead, you now need to handle the possible exception.